translations/ui-tenant-settings/en.json

```diff
@@ -193,5 +193,6 @@
   "permission.settings.sso": "Settings (tenant): Can maintain SSO settings",
   "permission.settings.location": "Settings (tenant): Can create, edit and remove locations",
   "permission.settings.servicepoints": "Settings (tenant):  Can create, edit and remove service points",
-  "permission.settings.bursar-exports": "Settings (tenant): Bursar admin"
+  "permission.settings.bursar-exports": "Settings (tenant): Bursar admin",
+  "settings.location.institutions.number": "{number}"
```

Comment:
Do we really need it here?
We have only the "number" parameter without any word(s) that can be translated. Looks like it is redundant.

Reply:
Agree, I removed that